src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery.Test/ScenarioTests/A2A/A2ATestsHelper.ps1

```diff
@@ -0,0 +1,200 @@
+  return getVaultLocation
+}
+
+function getPrimaryFabric{
+    return  "a2aPrimaryFabric"+$seed
+
+}
+
+function getRecoveryFabric{
+    return  "a2aRecoveryFabric"+$seed
+
+}
+
+function getAzureVm{
+    param([string]$primaryLocation)
+    
+        $VMLocalAdminUser = "adminUser"
+        $VMLocalAdminSecurePassword = "password"
+        $password=$VMLocalAdminSecurePassword|ConvertTo-SecureString -AsPlainText -Force
+        $Credential = New-Object System.Management.Automation.PSCredential ($VMLocalAdminUser, $password);
+        New-AzureRmVM -Name MyVm -Credential $Credential -location getPrimaryLocation
+}
+
+function getPrimaryPolicy{
+    return "TestA2APolicy1" + $seed;
+}
+
+function getRecoveryPolicy{
+    return "TestA2APolicy1" + $seed;
+}
+
+function getPrimaryContainer{
+    return "A2APrimaryContainer"+ $seed;
+}
+
+
+function getRecoveryContainer{
+    return "A2ARecoveryContainer"+ $seed;
+}
+
+
+function getPrimaryContainerMapping{
+    return "A2APCM"+ $seed;
+}
+
+
+function getRecoveryContainerMapping{
+    return "A2ARCM"+ $seed;
+}
+
+function getPrimaryNetworkMapping{
+    return "A2ANetworkMapping"+ $seed;
+}
+
+function getRecoveryNetworkMapping{
+    return "A2ARecoveryNetworkMapping"+ $seed;
+}
+
+function getPrimaryNetworkId{
+    param([string] $location , [string] $resourceGroup)
+
+    
+}
+
+function getRecoveryNetworkId{
+    param([string] $location , [string] $resourceGroup)
+
+    $primaryNetworkName = "recoveryNetwork"+ $location + $seed;
+    $virtualNetwork = New-AzureRmVirtualNetwork `
+          -ResourceGroupName $resourceGroup `
+          -Location $location `
+          -Name $primaryNetworkName `
+          -AddressPrefix 10.0.0.0/16
+    $virtualNetwork.id
+}
+
+
+##
+
+<#
+.SYNOPSIS
+Wait for job completion
+Usage:
+    WaitForJobCompletion -JobId $Job.ID
+    WaitForJobCompletion -JobId $Job.ID -NumOfSecondsToWait 10
+#>
+function WaitForJobCompletion
+{ 
+    param(
+        [string] $JobId,
+        [int] $JobQueryWaitTimeInSeconds = 20,
+        [string] $Message = "NA"
+        )
+        $isJobLeftForProcessing = $true;
+        do
+        {
+            $Job = Get-AzureRmRecoveryServicesAsrJob -Name $JobId
+            Write-Host $("Job Status:") -ForegroundColor Green
+            $Job
+
+            $isJobLeftForProcessing = ($Job.State -eq 'InProgress' -or $Job.State -eq 'NotStarted')
+            
+            if($isJobLeftForProcessing)
+            {
+                if($Message -ne "NA")
+                {
+                    Write-Host $Message -ForegroundColor Yellow
+                }
+                else
+                {
+                    Write-Host $($($Job.JobType) + " in Progress...") -ForegroundColor Yellow
+                }
+                Write-Host $("Waiting for: " + $JobQueryWaitTimeInSeconds.ToString + " Seconds") -ForegroundColor Yellow
+                [Microsoft.Azure.Test.TestUtilities]::Wait($JobQueryWaitTimeInSeconds * 1000)
```

Comment:
Should be using `[Microsoft.Rest.ClientRuntime.Azure.TestFramework.TestUtilities]::Wait`